content/en/docs/tasks/debug/debug-cluster/kubectl-node-debug.md

```diff
@@ -47,22 +50,26 @@ of the Node will be mounted at `/host`
 
 When creating a debugging session on a Node, keep in mind that:
 
-* `kubectl debug` automatically generates the name of the new pod based on
+* `kubectl debug` automatically generates the name of the new pod, based on
   the name of the node.
 * The root filesystem of the Node will be mounted at `/host`.
-* The container runs in the host IPC, Network, and PID namespaces, although
-  the pod isn't privileged, so reading some process information may fail,
-  and `chroot /host` will fail.
+* Although the container runs in the host IPC, Network, and PID namespaces,
+  the pod isn't privileged. This means that reading some process information might fail
+  because access to that information is restricted to superusers. `chroot /host` will fail.
 * If you need a privileged pod, create it manually.
```

Comment:
```suggestion
  because access to that information is restricted to superusers. `chroot /host` will fail.
  If you need a privileged pod, create it manually.
```

The two points are related so I recommend that you merge them :)